source/base/Promise.ooc

```diff
@@ -103,6 +104,13 @@ _ThreadResultPromise: class <T> extends ResultPromise<T> {
 		status := this wait()
 		status ? this _result[T] : defaultValue
 	}
+	getResult: func (defaultValue: T) -> T {
+		status: Bool
+		this _mutex lock()
+		status = (this _state == _PromiseState Finished)
```

Comment:
This is still a legit comment. Why not `status := (this _state == _PromiseState Finished)`?